### PR TITLE
Xuejun/remove apis version

### PR DESCRIPTION
**Pull request by zhaixuejun1993**
### Details:
 - Deprecated `ov::DiscreteTypeInfo` member `version`
 - Remove related reference or using about `version`

### Tickets:
 - 100091

### CODE REVIEW

File: src/bindings/python/src/compatibility/pyngraph/node.cpp [hunk 1]
```diff
@@ -278,16 +278,6 @@ void regclass_pyngraph_Node(py::module m) {
                 get_rt_info : PyRTMap
                     A dictionary of user defined data.
              )");
-    node.def("get_version",
```
Comment (ilya-lavrenov) [inline src/bindings/python/src/compatibility/pyngraph/node.cpp:281]: this API is not deprecated in python
@ilyachur do we need to provide alternative solution for BW compatibility? E.g. parse `string` to `int` and return `int` as it was before

File: src/common/low_precision_transformations/include/low_precision/rt_info/avg_pool_precision_preserved_attribute.hpp [hunk 2]
```diff
@@ -24,7 +24,7 @@ namespace ngraph {
  */
 class LP_TRANSFORMATIONS_API AvgPoolPrecisionPreservedAttribute : public PrecisionPreservedAttribute {
 public:
-    OPENVINO_RTTI("LowPrecision::AvgPoolPrecisionPreserved", "", ov::RuntimeAttribute, 0);
+    OPENVINO_RTTI("LowPrecision::AvgPoolPrecisionPreserved", "", ov::RuntimeAttribute);
```
Comment (riverlijunjie) [inline src/common/low_precision_transformations/include/low_precision/rt_info/avg_pool_precision_preserved_attribute.hpp:27]: `version` is deprecated, and should be replaced by `version_id`, does it make sense?

Comment (zhaixuejun1993) [reply src/common/low_precision_transformations/include/low_precision/rt_info/avg_pool_precision_preserved_attribute.hpp:27]: > `version` is deprecated, and should be replaced by `version_id`, does it make sense?

The second parameter is `version_id`.

**Comment by zhaixuejun1993** [comment]
Caused by this changes, the other components need change too. Pls help review the following PR:
ARM Plugin: [598](https://github.com/openvinotoolkit/openvino_contrib/pull/598)
VPUX Plugin: [3489](https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/3489)

File: src/bindings/python/src/compatibility/pyngraph/node.cpp (hunk 1 above)
Comment (ilyachur) [reply src/bindings/python/src/compatibility/pyngraph/node.cpp:281]: We can return opset version

**Comment by zhaixuejun1993** [comment]
Hi @ilyachur @ilya-lavrenov  @riverlijunjie Pls help review & merge this PR. 
This PR works depend on the openvino_contrib PR: [608](https://github.com/openvinotoolkit/openvino_contrib/pull/608)
Here is the CI build & test result (combine this PR-15773 and PR-608): [CI PASS](https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1959)
**Pls make merge the two PRs at the same time.**

**Comment by ilyachur** [comment]
> Hi @ilyachur @ilya-lavrenov @riverlijunjie Pls help review & merge this PR. This PR works depend on the openvino_contrib PR: [608](https://github.com/openvinotoolkit/openvino_contrib/pull/608) Here is the CI build & test result (combine this PR-15773 and PR-608): [CI PASS](https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/1959) **Pls make merge the two PRs at the same time.**

I took a look, and looks like we can merge these PRs step by step. I don't see the reason why PR 608 should depend on these changes. We can fix it by using opset name instead of int version

**Comment by zhaixuejun1993** [comment]
Replaced by PR: https://github.com/openvinotoolkit/openvino/pull/16426

**Pull request closed without merging**